### PR TITLE
feat: add forum link to collections

### PR DESCRIPTION
**Pull request by nicosantangelo**

### CODE REVIEW

File: src/Forum/Forum.router.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// import { Request, Response } from 'express'
```
Comment (nachomazzara) [inline src/Forum/Forum.router.ts:1]: Oops!

File: src/Forum/client.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+export type CreateError = {
+  action: string
+  errors: string[]
+}
+
+type CreateResponse = CreateSuccess | CreateError
+
+export const FORUM_URL = env.get('REACT_APP_FORUM_URL', '')
```
Comment (nachomazzara) [inline src/Forum/client.ts:23]: Is this variable being used outside this file?

File: src/Forum/client.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import fetch, { Response } from 'node-fetch'
+import { env } from 'decentraland-commons'
+import { ForumPost } from './Forum.types'
+
+export type CreateSuccess = {
```
Comment (nachomazzara) [inline src/Forum/client.ts:5]: Should we need to keep these types here or on `Forum.types.ts`?

File: src/Forum/client.ts (hunk 2 above)
Comment (nicosantangelo) [reply src/Forum/client.ts:23]: Not really! I'll delete the export

File: src/Forum/client.ts (hunk 3 above)
Comment (nicosantangelo) [reply src/Forum/client.ts:5]: Good catch, probably on `.types.ts`, I meant to have them there and forgot

**Pull request merged**